src/App.js

```diff
@@ -9,13 +9,26 @@ class App extends React.Component {
   constructor(){
     super()
     this.state={
-      data: movieData,
+      data: movieData.movies,
       currentView: "main",
       error: "",
       selectedMovie: 0
     };
   };
 
+  componentDidMount = () => {
+    return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+        .then ( response => response.json())
+        .then (movies => this.updateMovieData(movies.movies))
+        .catch(error => alert('Something went wrong.  Please try again later'));
```

Comment:
Overall nice work setting up your `componentDidMount`.  I would spend a bit more time with error handling including checking that the `response.ok` is good before parsing the response and **throwing a new Error** if not.  Also instead of firing an `alert`, try updating your state with the new error message!